packages/web/pages/index.tsx

```diff
@@ -43,11 +44,13 @@ const Home = () => {
         <div className="absolute inset-0 top-[104px] flex h-auto w-full justify-center md:top-0">
           <div className="flex w-[512px] flex-col gap-4 lg:mx-auto md:mt-5 md:w-full md:px-5">
             {featureFlags.swapsAdBanner && <SwapAdsBanner />}
-            <TradeTool
-              page="Swap Page"
-              previousTrade={previousTrade}
-              setPreviousTrade={setPreviousTrade}
-            />
+            <ClientOnly>
+              <TradeTool
+                page="Swap Page"
+                previousTrade={previousTrade}
+                setPreviousTrade={setPreviousTrade}
+              />
+            </ClientOnly>
```

Comment:
Could you provide some context on why this is needed now? Maybe a comment would be great too

Reply:
I'm not sure. It was originally wrapping within TradeTool component, but I moved it back to here since I started having intermittent hydration errors in dev server. (I believe the errors are silenced in prod). I didn't want to take the time now to debug